src/Relation/Binary/Lattice.agda

```diff
@@ -193,6 +215,39 @@ record Lattice c ℓ₁ ℓ₂ : Set (suc (c ⊔ ℓ₁ ⊔ ℓ₂)) where
 
   open JoinSemilattice joinSemilattice public using (poset; preorder)
 
+record IsDistributiveLattice {a ℓ₁ ℓ₂} {A : Set a}
+                             (_≈_ : Rel A ℓ₁) -- The underlying equality.
+                             (_≤_ : Rel A ℓ₂) -- The partial order.
+                             (_∨_ : Op₂ A)    -- The join operation.
+                             (_∧_ : Op₂ A)    -- The meet operation.
+                             : Set (a ⊔ ℓ₁ ⊔ ℓ₂) where
+  field
+    isLattice    : IsLattice _≈_ _≤_ _∨_ _∧_
+    ∧-∨-distribˡ : _DistributesOverˡ_ _≈_ _∧_ _∨_
```

Comment:
The convention for naming distributive properties is `∧-distribˡ-∨`. Please could they be changed throughout? Thanks! `Data.Nat.Properties` is always a good place to check if you're not sure.

Reply:
I see. I copied this from `Algebra.Structures`.

Reply:
Good spot, that needs to be changed.